src/components/forms/Login.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { gql } from 'apollo-boost';
+import { useQuery, useMutation } from 'react-apollo';
+
+const LOGIN_USER = gql`
+    query loginUser($email: String!, $password: String!){
+        loginUser(email: $email, password: $password){
+            email
+            password
+        }
+    }
+`
+
+
+const Login = () => {
+
+    const [ email, setEmail ] = useState('');
+    const [ password, setPassword ] = useState('');
+
+    const [ loginUser, {data, loading, error} ] = useMutation(LOGIN_USER);
+
+    const onSubmit = e => {
+        e.preventDefault();
+        loginUser({ variables: {email, password} })
+    };
+
+    return (
+        <>
+          <form onSubmit={onSubmit()}>
```

Comment:
On this line (29) I believe only the function reference should be provided and it should not be invoked.

So, it would look like `<form onSubmit={onSubmit}>` instead.